### PR TITLE
capa explorer: updating supported IDA versions

### LINKED ISSUE #845: IDA Plugin Doesn't Support IDA 7.7
### Description

The most recent release v3.0.3 is missing support for IDA 7.7 in the file C:\<python_version>\Lib\site-packages\capa\capa\ida\helpers\__init__.py.

### Steps to Reproduce
1. Install via pip: `pip install flare-capa`
2. Copy `capa_explorer.py` from Github to IDA `plugins` folder
3. Open an executable in IDA 7.7

**Expected behavior:**

Be able to launch capa explorer plugin via `Edit->Plugins -> FLARE capa explorer`.

**Actual behavior:**

Message in IDA output specifying IDA 7.7 is not a supported version:

`WARNING:capa:This plugin does not support your IDA Pro version`
`WARNING:capa:Your IDA Pro version is: 7.7. Supported versions are: 7.4, 7.5, 7.6.`

### Versions

Running Windows 11 x64 with Python 3.9.6 and Windows 10 x64 Python 3.9.6. Installed capa via pip (`pip install flare-capa`). Using IDA Pro integration (`capa explorer`).

`capa --version`
- capa 3.0.3

### Additional Information
N/A

### PR DESCRIPTION
**Pull request by mike-hunhoff**
fixes #845. pinning supported IDA version to IDA >= 7.4 and IDA < 8.0.

### CODE REVIEW

**Comment by mike-hunhoff** [comment]
TLDR; tested Windows 10; Python 3.9.9, IDA 7.7

**Comment by williballenthin** [comment]
lgtm 

i’d say we should do a release, though we have some non-trivial enhancements too (perf fixes), so maybe a minor release?

**Comment by mike-hunhoff** [comment]
@williballenthin minor release sounds good to me.
